R/dvhydrograph-render.R

```diff
@@ -46,7 +46,7 @@ createDVHydrographPlot <- function(reportObject){
 
   #Find the highest priority TS that has data
   priorityTS <- list(stat1TimeSeries, stat2TimeSeries, stat3TimeSeries, stat1TimeSeriesEst, stat2TimeSeriesEst, stat3TimeSeriesEst)
-  priorityTS <- priorityTS[[1]]
+  priorityTS <-  priorityTS <- priorityTS[unlist(lapply(priorityTS, function(ts){!isEmptyOrBlank(ts)}))][[1]]
```

Comment:
Is the double assignment necessary? Just looks odd doing blah <- blah <- (stuff done to blah). 
Can we not do blah <- (stuff done to blah)?